Source/SIMPLView/SIMPLView_UI.cpp

```diff
@@ -966,9 +966,25 @@ void SIMPLView_UI::processPipelineMessage(const PipelineMessage& msg)
       }
     }
 
-    if(stdOutDockWidget->isVisible() == false)
+    // Allow status messages to open the standard output widget
+    if(HideDockSetting::OnStatusAndError == m_HideStdOutput)
     {
-      stdOutDockWidget->toggleViewAction()->toggle();
+      stdOutDockWidget->setVisible(true);
+
+#if 0
```

Comment:
Do we need this section of code? Since it is not compiled can we just take it out or are we leaving it in for historical or documentation reasons?